content/millennium/dstu2/infrastructure/binary.md

```diff
@@ -29,20 +31,21 @@ List an individual Binary by its id:
 
 _Implementation Notes_
 
-* This is usually linked from DiagnosticReport and should generally be accessed using the exact link given in that resource. Modifying the link has undefined consequences.
+* This is usually linked from DocumentReference or DiagnosticReport and should generally be accessed using the exact link given in the referring resource. Modifying the link has undefined consequences.
+* Documents containing URL sections are unsupported with contentType of `application/pdf`
 * See the [headers](#headers) section for concerns about the Accept header.
 * See the [authorization](#authorization-types) section for concerns about the required OAuth scopes.
 
 ### Authorization Types
 
-Requires both the appropriate Binary.read and DiagnosticReport.read scopes to be granted to the caller.
+The necessary OAuth scopes can be determined by checking the attachment.contentType in the referring resource. For attachments with the `application/pdf` contentType, the Binary.read and DocumentReference.read scopes are required. For attachments with the `text/html` contentType, the Binary.read and DiagnosticReport.read scopes are required.
```

Comment:
The scopes might need to change now.

Reply:
The scopes can't be updated until after a local service change coming out at the end of the month.